keras_nlp/layers/preprocessing/masked_lm_mask_generator.py

```diff
@@ -170,11 +170,7 @@ def __init__(
     def call(self, inputs):
         inputs, unbatched, rectangular = convert_to_ragged_batch(inputs)
 
-        (
-            token_ids,
-            mask_positions,
-            mask_ids,
-        ) = tf_text.mask_language_model(
+        (token_ids, mask_positions, mask_ids,) = tf_text.mask_language_model(
```

Comment:
this looks like you are using an older version of black maybe?

if you remove the trailing comma it would reformat to one line

Reply:
Fixed! Yeah just needed to update black